packages/coding-components/templates/angular/src/app/app.module.ts

```diff
@@ -13,20 +13,35 @@
  * limitations under the License.
  */
 
-import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-  // Must be included for component packages that don't yet have an Angular wrapper
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { APP_INITIALIZER, NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+// Must be included for component packages that don't yet have an Angular wrapper
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
```

Comment:
I originally included this because Calcite didn't have an Angular wrapper at the time. This should no longer be necessary if we include [ `@esri/calcite-components-angular`](https://www.npmjs.com/package/@esri/calcite-components-angular) in our `package.json`